src/containers/org/members/MembersSection.js

```diff
@@ -157,7 +168,9 @@ const MembersSection = ({
           const userId = getUserId(member);
           return (
             <MemberWrapper key={userId}>
-              <span>{getUserProfileLabel(member, thisUserId)}</span>
+              <span data-user-id={userId} onClick={goToMember} onKeyPress={goToMember} role="link" tabIndex={0}>
```

Comment:
Doesn't seem like this is achieving anything a regular <a> tag isn't also meant for. Also does middle clicking a role=link open a new tab?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role

Reply:
Great points. I can change it to `role="button"`. The reason for this in the first place was because of eslint warnings about accessibility.

Reply:
i'm really suggesting to use an `<a>` tag like with `<Link>` instead. why go through the trouble of making a fake link when you can use a real one.

You can get rid of `goToMember` to avoid the `.find()` it executes on click even though you already know who the member is when the link is rendered through `pageMembers.map()`

```
{
  pageMembers.map((member) => {
    const userId = getUserId(member);
    const memberPrincipalId :?UUID = getSecurablePrincipalId(member);
    const memberPath = Routes.ORG_MEMBER
      .replace(Routes.ORG_ID_PARAM, organizationId)
      .replace(Routes.PRINCIPAL_ID_PARAM, memberPrincipalId);

    return (
      <MemberWrapper key={userId}>
        <Link to={memberPath}>{getUserProfileLabel(member, thisUserId)}</Link>
        <IconButton onClick={() => handleOnClickRemoveMember(member)}>
          <FontAwesomeIcon fixedWidth icon={faTimes} />
        </IconButton>
      </MemberWrapper>
    );
  })
}
```

Reply:
Wow. This is so much better. Thanks. Updated!